Tests/Opc.Ua.Core.Tests/Stack/Server/ServerBaseTests.cs

```diff
@@ -224,19 +224,19 @@ public void FilterByClientUrlTest(string endpointUrl, int baseAddressCount = Bas
         /// </summary>
         [Test]
         [TestCase("opc.tcp://localhost:51210/UA/SampleServer")]
-        [TestCase("opc.tcp://externalhostname.com:50001/UA/SampleServer", 6)]
-        [TestCase("opc.tcp://externalhostname.com:52541/UA/SampleServer", 6)]
+        [TestCase("opc.tcp://externalhostname.com:50001/UA/SampleServer", 0)]
+        [TestCase("opc.tcp://externalhostname.com:52541/UA/SampleServer", 0)]
         // [TestCase("opc.tcp://[fe80:1234::8]:51210/UA/SampleServer", 4)]
         // [TestCase("opc.tcp://[fe80:1234::8%3]:51210/UA/SampleServer", 4)]
         [TestCase("opc.tcp://[2003:d9:1f0c:5e00:4139:ee31:6cc3:313e]:62541/UA/SampleServer", 6)]
         [TestCase("opc.tcp://myhostname.com:51210/UA/SampleServer", 6)]
         [TestCase("opc.tcp://UNKNOWNHOSTNAME.COM:51210/UA/SampleServer", 4)]
-        [TestCase("opc.tcp://EXTERNALHOSTNAME.COM:51210/UA/SampleServer", 6)]
+        [TestCase("opc.tcp://EXTERNALHOSTNAME.COM:51210/UA/SampleServer", 0)]
         [TestCase("opc.tcp://localhost:51210/UA/SampleServer")]
```

Comment:
the cases which try to access using a configured AlternateBaseUrl are again failing, because there is no simple way to match them against an internal endpoint. We need to document that alternateBaseUrl with different port always return no matching endpoint, so the configuration should not be used.